nasa-gibs/worldview
# WV-2709 modulate transition speed if in kiosk mode

## Description
We recently did a global speed reduction on transitions for events/tour stories. We would like to restore the original speed if NOT in kiosk mode & retain the longer duration transitions when IN kiosk mode.

Fixes # wv-2709

## How To Test
- git checkout wv-2709-modulate-transition-speed
- npm run watch
- Test [NOT in kiosk mode](http://localhost:3000/?lg=false&t=2023-05-03-T14%3A48%3A29Z) - confirm speed of transition is faster than the production site.
- Test [in kiosk mode ](http://localhost:3000/?kiosk=true&lg=false&t=2023-05-03-T14%3A48%3A29Z) - confirm speed of transition is slower (about 50%) than the production site.

Note: Click the Information ("i") icon on the top right & select "Explore" to bring up the tours.
Also Note: This PR is branched from WV-2669 as that branch has the logic for kiosk mode (which is not yet in develop)

@nasa-gibs/worldview
